pkg/os/volume/api.go

```diff
@@ -41,6 +41,8 @@ type VolumeAPI struct{}
 // verifies that the API is implemented
 var _ API = &VolumeAPI{}
 
+const minimum_extent_size = 1024 * 1024
```

Comment:
Thanks for adding an integration tests, sorry to be annoying but this should be `MinimumExtentSize`, also, is `expand` a better word for it?

Reply:
done.